test/functional/helpers.lua

```diff
@@ -411,6 +411,7 @@ end
 function module.clear(...)
   local argv, env = module.new_argv(...)
   module.set_session(module.spawn(argv, nil, env))
+  module.request('nvim_set_client_info', 'testclient', {}, 'remote', {}, {})
```

Comment:
@justinmk, can this line be moved to the `log_spec.lua` file? This has been interfering with the `encoding_spec.lua` test. I'm working on this here: https://github.com/neovim/neovim/pull/12503.

Reply:
sure, don't let it block you. Though the intention was to make it a part of all tests, since every Nvim client should set client-info (and the test client is a Nvim client)